incubator/sparkoperator/templates/spark-operator-rbac.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: sparkoperator
+rules:
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["*"]
+- apiGroups: [""]
+  resources: ["services", "configmaps"]
```

Comment:
I will experiment adding here secrets, not sure if this helps

Reply:
@yuchaoran2011 maybe I am mistaken, but we are creating with `sparkoperator` token a a secret, but here we don't allow creating secrets? How is the batch job able to create it ?

Reply:
@yuchaoran2011 you HAVE to add here `secrets` resources for this clusterrole